src/sql/workbench/services/profiler/browser/profilerService.ts

```diff
@@ -317,6 +318,18 @@ export class ProfilerService implements IProfilerService {
 		if (fileURIs?.length === 1) {
 			const fileURI = fileURIs[0];
 
+			try {
+				let fileSize = await (await fs.stat(fileURI.fsPath)).size;
+				console.log(fileSize);
+				const fileLimitSize = 1073741824; 	// 1GB
+				if (fileSize > fileLimitSize) {
+					this._notificationService.error(nls.localize('FileTooLarge', "File is too large to open and it may hang ADS. "));
```

Comment:
is this similar to the message that's displayed when opening other files that are too big in ADS? Would be good for the strings to be the same

Reply:
These are the strings used at a few of the places:
"File is too large to open as untitled editor. Please upload it first into the file explorer and then try again."
"Unable to read file '{0}' that is too large to open"
"File is too large to open"
"At least one file is not displayed in the text compare editor because it is very large ({0})."
"The file is not displayed in the text editor because it is very large ({0})."

Does this string sound better: "The file is not displayed in the profiler because it is very large ({0})." ?

Reply:
Maybe "File is too large to open in profiler"? Do any of those other places also state the max file size?

Reply:
No, none of the places mention the max file size. I will update it to "The file is too large to open in profiler".

Reply:
Updated it as discussed.